### DISCUSSION TITLE
alias for "!kc lunar chest"

### DISCUSSION THREAD

**Discussion by ingmarins** [runelite/runelite#17711] [Ideas]
suggestions for lunar chest kc command alias

moon
moons
moons of peril
perilous moons

**Comment by Nightfirecat** [comment]
Duplicate of #17699, this is already done and will be available with our next release.

**Reply by ingmarins** [reply]
sorry, thats a PEBCAK on my part, looked for discussions and open issues, but not closed issues. thank you, looking forward to this!
